rtsp.go

```diff
@@ -599,43 +615,59 @@ func (rc *rtspCamera) initH265(session *description.Session) (err error) {
 			return
 		}
 
-		// If the AU has more than one NALU, compact them into a single payload with NALUs separated
-		// in AnnexB format. This is necessary because the H.265 decoder expects all NALUs for a frame
-		// to be in a single payload rather than chunked across multiple decode calls.
-		packed := []byte{}
-		firstNALU := true
-		for _, nalu := range au {
-			if len(nalu) == 0 {
-				rc.logger.Warn("empty NALU found in H265 AU, skipping NALU")
-				continue
-			}
-			// Add start code prefix to all NALUs except the first non-empty one.
-			if !firstNALU {
-				packed = append(packed, H2645StartCode()...)
+		packedAU := packH265AUIntoNALU(au, rc.logger)
```

Comment:
https://github.com/viam-modules/viamrtsp/pull/82 fixed the incompatibility with the link 4k camera by compacting multiple NALUs into a single one. 

This preserves that behavior both for eagerly decoded packets and lazily decoded packets.